### PR TITLE
en, zh: update tkctl debug usage

### PR DESCRIPTION
**Pull request by DanielZhangQD**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: en/use-tkctl.md [hunk 1]
```diff
@@ -218,11 +218,12 @@ local-pv-e54c122a   pd-demo-cluster-pd-2       Bound    1476Gi     172.16.4.156
...
+This command is used to diagnose the Pods in a TiDB cluster. It launches a debug launcher Pod and then the Pod launches a container with the specified docker image on the host that holds the target Pod. The container has the necessary troubleshooting tools installed and shares the namespace with the container in the target Pod, so you can seamlessly diagnose the target container by using various tools in the debug container.
 
 | Flag | Abbreviation | Description |
 | ----- | --------- | ----------- |
 | --image |    | Specify the docker image used by the debug container; default to `pingcap/tidb-debug:lastest` |
+| --launcher-image         |           | Specify the docker image for the debug launcher pod which is responsible to launch the debug container; default to `pingcap/debug-launcher:latest`                |
```
Comment (ran-huang) [inline en/use-tkctl.md:226]:
```suggestion
| --launcher-image         |           | Specify the docker image for the debug launcher pod which is responsible for launching the debug container; default to `pingcap/debug-launcher:latest`                |
```

**Review by ran-huang — APPROVED** [review]

File: en/use-tkctl.md [hunk 2]
```diff
@@ -218,11 +218,12 @@ local-pv-e54c122a   pd-demo-cluster-pd-2       Bound    1476Gi     172.16.4.156
 
 ### tkctl debug [pod_name]
 
-This command is used to diagnose the Pods in a TiDB cluster. It launches a debug container with the specified docker image on the host that holds the target Pod. The container has the necessary troubleshooting tools installed and shares the namespace with the container in the target Pod, so you can seamlessly diagnose the target container by using various tools in the debug container.
+This command is used to diagnose the Pods in a TiDB cluster. It launches a debug launcher Pod and then the Pod launches a container with the specified docker image on the host that holds the target Pod. The container has the necessary troubleshooting tools installed and shares the namespace with the container in the target Pod, so you can seamlessly diagnose the target container by using various tools in the debug container.
```
Comment (dragonly) [inline en/use-tkctl.md:221]:
```suggestion
This command is used to diagnose the Pods in a TiDB cluster. It launches a debug launcher Pod which then starts a debug container using specified docker image on the same host of the target Pod. The container has necessary debug tools and shares the same namespace with the container in the target Pod, so you can diagnose the target container by using debug tools unaware of the debug container.
```

Comment (dragonly) [reply en/use-tkctl.md:221]: rest LGTM

Comment (DanielZhangQD) [reply en/use-tkctl.md:221]:
```suggestion
This command is used to diagnose the Pods in a TiDB cluster. It launches a debug launcher Pod which then starts a debug container using the specified docker image on the same host of the target Pod. The container has necessary troubleshooting tools and shares the same namespaces with the container in the target Pod, so you can diagnose the target container by using various tools in the debug container.
```

**Comment by dragonly** [comment]
LGTM

**Pull request merged**